tracing/opentracing/src/main/java/io/helidon/tracing/opentracing/OpenTracingSpan.java

```diff
@@ -91,6 +94,16 @@ public Scope activate() {
         return new OpenTracingScope(tracer.activateSpan(delegate));
     }
 
+    @Override
+    public Span baggage(String key, String value) {
+        return (OpenTracingSpan) delegate.setBaggageItem(key, value);
+    }
+
+    @Override
+    public String baggage(String key) {
```

Comment:
Our APIs can never return `null`, this should return an `Optional<String>` - and the "noop" should return an empty optional

Reply:
Done